README.md

```diff
@@ -1,24 +1,189 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Users can create an account and sign in to ensure their data remains confidential.**
+- **They can add new transactions linked to specific categories.**
+- **Users have the ability to view their expenditure breakdown for each category.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-app-9h5y.onrender.com/)
+- [Video presentation](https://www.loom.com/share/dc26c30f4c1d41f6b9b42011bfe114b2?sid=4c6528c9-1c58-4a05-bbbb-5cb3dac63018)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- `Linux Operating system`
+- `Text Editor (VSCode, RubyMine)`
+- `Ruby`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/AbdelazizSoliman/Budget_app.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec
+```
+
+### Deployment
+
+You can deploy this project using your own deployment service like `Render`.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Abdelaziz Soliman**
+
+- GitHub: [Abdelaziz](https://github.com/AbdelazizSoliman)
+- LinkedIn: [Abdelaziz](https://www.linkedin.com/in/abdelazizsoliman)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Edit purchases**
+- [ ] **Improve styling**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project consider adding a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank Microverse.
+- Original design by Grégoire Vella, available on Behance.
```

Comment:
- [ ] Good job 👏 . But kindly add the highlighted link to the name. This is an important skill in the future, to avoid legal battles regarding the use of original content in your project. 

### The way it should look:
- [ ] Original design idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella).

### How to use in mackup:
`Original design idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella).`